portfolio/src/main/webapp/index.html

```diff
@@ -22,43 +22,43 @@
         }
     </style>
   </head>
-  <body style="background-color: #E6E6FA;" onload="getMessages()">
+  <body style="background-color: #E6E6FA;" onload="formSettingOnLoad()">
     <div id="content" style="text-align:center">
       <h1>Chia-Jou's Portfolio</h1>
       <p>This is Chia-Jou's portfolio.</p>
       <img src="/images/Chia-Jou Kuo.jpg" width="280">
       <span style="text-align: center; display:block">
-          <a href="https://www.linkedin.com/in/chia-jou-kuo-b77847175/">Linkedin</a>
-          <a href="https://github.com/chiajoukuo">GitHub</a>
+				<a href="https://www.linkedin.com/in/chia-jou-kuo-b77847175/">Linkedin</a>
+				<a href="https://github.com/chiajoukuo">GitHub</a>
       </span>
       <h2>Projects I've worked on.</h2>
-        <div style="margin-top:20px;" class="row">
-            <div class="column">
-                <img src="/images/pose.png" width="250" style="margin:0px 50px;">
-                <h3>Machine Learning</h3>
-                <h4>Hand-drawn Character Image Generation</h4>
-            </div>
-            <div class="column">
-                <img src="/images/endo.jpg" width="250" style="margin:0px 50px;">
-                <h3>Robotic</h3>
-                <h4>Endodontic Therapy Robot</h4>
-            </div>
-            <div class="column">
-                <img src="/images/pepper.PNG" width="250" style="margin:0px 50px;">
-                <h3>Robotic</h3>
-                <h4>Pose Imitating Assistant Robot</h4>
-            </div>
-        </div>
-<!--       
-      <p>Click here to get a random message:</p>
-      <button onclick="getMessages()">Hello</button> -->
+			<div style="margin-top:20px;" class="row">
+				<div class="column">
+						<img src="/images/pose.png" width="250" style="margin:0px 50px;">
+						<h3>Machine Learning</h3>
+						<h4>Hand-drawn Character Image Generation</h4>
+				</div>
+				<div class="column">
+						<img src="/images/endo.jpg" width="250" style="margin:0px 50px;">
+						<h3>Robotic</h3>
+						<h4>Endodontic Therapy Robot</h4>
+				</div>
+				<div class="column">
+						<img src="/images/pepper.PNG" width="250" style="margin:0px 50px;">
+						<h3>Robotic</h3>
+						<h4>Pose Imitating Assistant Robot</h4>
+				</div>
+			</div>
```

Comment:
nit: Indentation. Since the preceding `<h2>` and the starting `<div>` are at the same level, they should be at the same indentation.